templates/repo/view_list.tmpl

```diff
@@ -65,7 +65,7 @@
 							{{$subJumpablePathName := $entry.GetSubJumpablePathName}}
 							{{$subJumpablePath := SubJumpablePath $subJumpablePathName}}
 							<span class="octicon octicon-file-directory"></span>
-							<a href="{{EscapePound $.TreeLink}}/{{EscapePound $subJumpablePathName}}">
+							<a href="{{EscapePound $.TreeLink}}/{{EscapePound $subJumpablePathName}}" title="{{$subJumpablePathName}}">
```

Comment:
maybe still `EscapePound` ?

Reply:
Since this is the title, I don't think it needs to be escaped since that's for URLs , right?

Reply:
The template library should automatically ensure that the `{{$subJumpablePathName}}` is appropriately encoded for attributes.